docs/content/how-to/hibernate-workers.md

```diff
@@ -0,0 +1,120 @@
+
+function annotate_nodes() {
+    MACHINES="$(oc get machines -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} -o name | wc -l)"
+    if [[ ${MACHINES} -le 0 ]];then
+        echo "There is not machines or machineSets in the Hosted ControlPlane namespace, exiting..."
+        echo "HC Namespace: ${HC_CLUSTER_NS}"
+        echo "HC Clusted Name: ${HC_CLUSTER_NAME}"
+        exit 1
+    fi 
+
+    echo "Annotating Nodes to avoid Draining"
+    oc annotate -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} machines --all "machine.cluster.x-k8s.io/exclude-node-draining="
+    echo "Nodes annotated!"
+}
+
+
+## Fill these variables first
+export KUBECONFIG=<KubeconfigPath>
+export HC_CLUSTER_NS=<HostedClusterNamespace>
+export HC_CLUSTER_NAME=<HostedClusterName>
+
+CHECK_NS="$(oc get ns -o name ${2})"
+if [[ -z "${CHECK_NS}" ]];then
+    echo "Namespace does not exists in the Management Cluster"
+    exit 1
+fi
+
+CHECK_HC="$(oc get hc -n ${HC_CLUSTER_NS} -o name ${3})"
+if [[ -z "${CHECK_HC}" ]];then
+    echo "HC ${3} does not exists in the namespace ${2} of the Management Cluster"
+    exit 1
+fi
+
+annotate_nodes
+```
+
+</details>
+
+- The next step it's basically scale down the NodePool associated to our Hosted Cluster. In order to identify it we need to execute an `oc get nodepool -n <HostedCluster Namespace>`, the one where you created the HostedCluster and NodePool on deployment time.
+- To perform the scale down, you just need to grab the NodePool name from the last command and execute this other one `oc scale nodepool/<NodePool Name> --namespace <HostedCluster Namespace> --replicas=0`
+
+
+<details>
+<summary> How to scale down the nodes programatically </summary>
+
+```bash
+#!/bin/bash
+
+function scale_down_pool() {
+
+    # Validated that the nodes in AWS Scale down instantly, they take sometime to dissapear inside of Openshift
+    # but the draining is avoided for sure
+    echo "Scalling down the nodes for ${HC_CLUSTER_NAME} cluster"
+    NODEPOOLS=$(oc get nodepools -n ${HC_CLUSTER_NS} -o=jsonpath='{.items[?(@.spec.clusterName=="'${HC_CLUSTER_NAME}'")].metadata.name}')
+    oc scale nodepool/${NODEPOOLS} --namespace ${HC_CLUSTER_NS} --replicas=0
+    echo "NodePool ${NODEPOOLS} scaled down!"
+}
+
+## Fill these variables first
+export KUBECONFIG=<KubeconfigPath>
+export HC_CLUSTER_NS=<HostedClusterNamespace>
+export HC_CLUSTER_NAME=<HostedClusterName>
+
+CHECK_NS="$(oc get ns -o name ${2})"
+if [[ -z "${CHECK_NS}" ]];then
+    echo "Namespace does not exists in the Management Cluster"
+    exit 1
+fi
+
+CHECK_HC="$(oc get hc -n ${HC_CLUSTER_NS} -o name ${3})"
+if [[ -z "${CHECK_HC}" ]];then
+    echo "HC ${3} does not exists in the namespace ${2} of the Management Cluster"
+    exit 1
+fi
+
+scale_down_nodepool
+```
+
+</details>
+
+After these steps, you will see how the (in the AWS case) instances will be terminated instantly, but Openshift will take some time until the nodes get deleted because of the default timeouts set on the platforms.
```

Comment:
I'm not sure what this means. The Node will be deleted by CAPI as soon and only when the underlying instance has gone away. -  [This is to prevent issues from happening with e.g. stateful](https://github.com/kubernetes-sigs/cluster-api/issues/2565)